editor/src/components/inspector/inpector-selectors.tsx

```diff
@@ -64,7 +65,10 @@ export function useComputedSizeRef(prop: 'width' | 'height'): { readonly current
       return null
     }
 
-    const localFrame = MetadataUtils.getFrameOrZeroRect(elementPath, metadata)
+    const localFrame =
+      nullIfInfinity(
+        MetadataUtils.findElementByElementPath(metadata, elementPath)?.nonRoundedGlobalFrame,
```

Comment:
If we're using the non-rounded frame here now, then we should rename the hook to `useNonRoundedComputedSizeRef`

Reply:
renamed it in https://github.com/concrete-utopia/utopia/pull/4390/commits/013797c6ac8f15fbc6fb292b6cd4228bb3bbadc7